best-practices/howto_readme.md

```diff
@@ -0,0 +1,68 @@
+# README.md
+
+All projects should have a `README.md` file. This file should try to answer the following questions:
+
+- What it is
+- External requirements
+- How to install it
+- How to use it
+- How to deploy it (if appropriate)
+- How to run the tests
+- Include relevant badges:
+    - [Travis](http://docs.travis-ci.com/user/status-images/)
+    - [Gem Version](http://badge.fury.io/for/rb)
+    - Coverage ([coveralls](https://coveralls.io/))
+    - [Gemnsasium](https://gemnasium.com/)
+    - [waffle](https://waffle.io/)
+
+
+## Example
+
+```
+[![Build Status](https://travis-ci.org/sul-dlss/name-of-the-project.svg?branch=master)](https://travis-ci.org/sul-dlss/name-of-the-project) [![Coverage Status](https://coveralls.io/repos/sul-dlss/name-of-the-project/badge.png)](https://coveralls.io/r/sul-dlss/name-of-the-project) [![Dependency Status](https://gemnasium.com/sul-dlss/name-of-the-project.svg)](https://gemnasium.com/sul-dlss/name-of-the-project) [![Gem Version](https://badge.fury.io/rb/name-of-the-project.svg)](http://badge.fury.io/rb/name-of-the-project)
+
+# Name of the Project
+
+This project provides an implementation of the [XKCD Random Number](https://xkcd.com/221/) generator. This random number generator is particularly useful when predictable random numbers can improve performance.
+
+## Requirements
+
+- Ruby 3.0
+- libxml 5
+
+## Installation
+
+Add this line to your Rails application's Gemfile
+
+\```ruby
```

Comment:
@ndushay suggested putting the example in block text and it seems avoid any escaping problems.

I think it's good to provide an example in-line to provide quick and easy access to a model readme. As you can see, I've also linked to a number of README files that are following this practice.

I think having template repositories is an interesting idea, but I'd argue out of scope for this proposal, which only attempted to document the consensus we seemed to have during a F2F meeting.